java/maven/nbproject/project.properties

```diff
@@ -34,7 +34,7 @@ test.config.stableBTD.excludes=\
 
 jnlp.indirect.files=maven-nblib/netbeans-eventspy.jar,maven-nblib/netbeans-cos.jar
 
-# requires nb.javac for compiling of tests
+# TODO remove after JDK 17 bump (code uses CompilationUnitTree#getModule)
 requires.nb.javac=true
```

Comment:
since gradle is going to be 17+, maybe we should do this here too. I saw some sig validation issues in the log when I tried this so I didn't really bother.  However that was before https://github.com/apache/netbeans/pull/7117, maybe I should give it another try.